### ISSUE TITLE
potion level's value is gamemode spectator

### ISSUE THREAD

**Issue by ditt0132** [KAIYOHUGO/to-bedrock#8]
"potion.potency.0": "selectWorld.gameMode.spectator.line2"
line 664, and that key in java are not exists!

**Comment by KAIYOHUGO** [comment]
So weird, I will investigate this tomorrow.

**Comment by KAIYOHUGO** [comment]
Good job mojang, empty string is a nice translation value.

```properties
# bedrock lang file
potion.potency.0=
```

```json5
// minecraft java lang file
{
  // ...
  "selectWorld.gameMode.spectator.line2": "",
  // ...
}

**Comment by KAIYOHUGO** [comment]
Fix in 8a84ea1.

The fixed version of [map.json](https://github.com/KAIYOHUGO/To-Bedrock/files/14519669/map.json) here.
